cmd/util/util.go

```diff
@@ -34,10 +34,12 @@ import (
 // writeIfNotFound returns true if the file was created and false if it already exists
 func WriteIfNotFound(path, templateName, templateValue string, data interface{}) bool {
 	// Make sure the directory exists
-	os.MkdirAll(filepath.Dir(path), 0700)
-
+	err := os.MkdirAll(filepath.Dir(path), 0700)
+	if err != nil {
+		log.Fatalf("error to create the file %s : %v", filepath.Dir(path), err)
```

Comment:
nit: should be "failed to ensure that path %q exists: %v"

Reply:
It was gone with the rebase 👍